### ISSUE TITLE
Without a config.ini, the setting page fails to load

### ISSUE THREAD

**Issue by Digital-Storm** [Gabisonfire/dashboard-q#12]
If the config.ini is missing new settings, you cannot save the new settings that are available in the web ui, such as titles.

**Comment by Gabisonfire** [comment]
Alright I see what happened there. I'll find a workaround. For now, I guess you know but i'll write it in case anyone else has the problem. Just add the setting manually in the config.ini file or delete you config.ini file and setup a new one with the ui.

**Comment by darkcloud784** [comment]
Could simply add a toggle for initial installation or update. That way you know its someone that just updated or installed it, and can check if the file exists and delete it and create a new one if it does. I may experiment with this and see if I can do it. I'm trying to learn php and your project has been one that has been teaching me alot.

**Comment by Gabisonfire** [comment]
See if you wanna do that. I think it should just write the settings wheter or not it exist instead of having a toggle. This will prevent this from happening everytime we had a new setting. I'll work on it but right now i'm on another project so It might take me a little longer than usual. Thank you! And don't hesitate if you have any question.

**Comment by Digital-Storm** [comment]
This was partially my fault too apparently. I didn't have my web directory configured with the proper permissions, so it was throwing errors that it couldn't write the config.ini file.

tldr
Check your folder perms.

**Comment by Gabisonfire** [comment]
Fixed in https://github.com/Gabisonfire/dashboard-q/commit/6c5d6756bd96c5445c4c7f7f51e1a445ff5454e1
